src/Extensions/Features/src/Microsoft.Extensions.Features.csproj

```diff
@@ -12,7 +12,7 @@ Microsoft.AspNetCore.Http.Features.FeatureCollection
     <IsAspNetCoreApp>true</IsAspNetCoreApp>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <PackageTags>aspnetcore</PackageTags>
-    <Nullable>enable</Nullable>
```

Comment:
Why is nullable being removed?

Reply:
Nullable is now turned on by default for all projects. Removing because it's redundent.